capstone/src/main/webapp/script.js

```diff
@@ -316,20 +306,28 @@ async function averagePrecinctSentiment(policePrecinct) {
     }
   }
   const sentimentAverage = sentimentCount/list.length;
-  return getSentimentColor(sentimentAverage);
+  return getSentimentColor(sentimentAverage, policePrecinct);
 }
 
-function getSentimentColor(averageFeelings) {
+function getSentimentColor(averageFeelings, policePrecinct) {
+  let precinctColor = '#228B22';
   if (Math.round(averageFeelings) == 5) {
-    return '#165B33';
+    precinctColor = '#165B33';
   } else if (Math.round(averageFeelings) == 4) {
-    return '#146B3A';
+    precinctColor = '#146B3A';
   } else if (Math.round(averageFeelings) == 3) {
-    return '#F8B229';
+    precinctColor = '#F8B229';
   } else if (Math.round(averageFeelings) == 2) {
-    return '#EA4630';
+    precinctColor = '#EA4630';
   } else {
-    return '#BB2528';
+    precinctColor = '#BB2528';
+  }
+  const precinctLayer = mapAndSelection.map;
+
+  if (policePrecinct!='Richmond'&& policePrecinct!='Southern') {
```

Comment:
Should we also store these precincts as constant variables that are shared across the codebase to guard against typos?

Reply:
I'm not sure if anyone uses them but me. But I will make a constant variable if I don't find a way to draw Richmond and Southern. line 327 is just a temp fix until I finish drawing all of the precincts. I've been having trouble with those.

Reply:
Thanks for the clarification, yea this is usually just to prevent the case where lets say, we accidentally add a line of code that says `policePrecent != 'Richmod'` and don't notice it. But if this is temporary, feel free to just leave as is.